narwhal/jenkins/jobs/SCS_pipeline/config.xml

```diff
@@ -0,0 +1,137 @@
+					<trim>false</trim>
+				</hudson.model.StringParameterDefinition>
+				<hudson.model.BooleanParameterDefinition>
+					<name>DEBUG_MODE</name>
+					<defaultValue>false</defaultValue>
+				</hudson.model.BooleanParameterDefinition>
+				<hudson.model.BooleanParameterDefinition>
+					<name>NO_PUSH</name>
+					<defaultValue>false</defaultValue>
+				</hudson.model.BooleanParameterDefinition>
+				<hudson.model.BooleanParameterDefinition>
+					<name>REMOVE_DXP_REPO</name>
+					<defaultValue>false</defaultValue>
+				</hudson.model.BooleanParameterDefinition>
+				<hudson.model.BooleanParameterDefinition>
+					<name>REMOVE_EE_REPO</name>
+					<defaultValue>false</defaultValue>
+				</hudson.model.BooleanParameterDefinition>
+				<hudson.model.BooleanParameterDefinition>
+					<name>REMOVE_LOG_DIR</name>
+					<defaultValue>false</defaultValue>
+				</hudson.model.BooleanParameterDefinition>
+				<hudson.model.StringParameterDefinition>
+					<name>GITHUB_PROJECT</name>
+					<description>The Github project you want to use</description>
+					<defaultValue>liferay</defaultValue>
+					<trim>false</trim>
+				</hudson.model.StringParameterDefinition>
+				<hudson.model.StringParameterDefinition>
+					<name>ZIP_LIST_RETENTION_TIME</name>
+					<defaultValue>1 min</defaultValue>
+					<trim>false</trim>
+				</hudson.model.StringParameterDefinition>
+			</parameterDefinitions>
+		</hudson.model.ParametersDefinitionProperty>
+		<org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+			<triggers>
+				<hudson.triggers.TimerTrigger>
+					<spec>H/10 * * * * </spec>
+				</hudson.triggers.TimerTrigger>
+			</triggers>
+		</org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+	</properties>
+	<definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2729.2732.vda_e3f07b_5a_f8">
+		<scm class="hudson.plugins.git.GitSCM" plugin="git@4.11.5">
+			<configVersion>2</configVersion>
+			<userRemoteConfigs>
+				<hudson.plugins.git.UserRemoteConfig>
+					<url>https://github.com/lucasmiranda0/liferay-docker.git</url>
+					<credentialsId>ci-hu</credentialsId>
+				</hudson.plugins.git.UserRemoteConfig>
+			</userRemoteConfigs>
+			<branches>
+				<hudson.plugins.git.BranchSpec>
+					<name>*/scs</name>
+				</hudson.plugins.git.BranchSpec>
+			</branches>
+			<doGenerateSubmoduleConfigurations>false</doGenerateSubmoduleConfigurations>
+			<submoduleCfg class="empty-list" />
+			<extensions/>
+		</scm>
+		<scriptPath>./narwhal/source_code_sharing/Jenkinsfile</scriptPath>
+		<lightweight>true</lightweight>
+	</definition>
+	<triggers/>
+	<disabled>false</disabled>
```

Comment:
@kiwm, can we sort this?

For example, `<definition>` and `<disabled>` are not following alphabetical order.